### DISCUSSION TITLE
Request - Step by step to migrate from SQL to Mariadb and move existing projects?

### DISCUSSION THREAD

**Discussion by KoolPal** [leokhoa/laragon#589] [Q&A] [answered]
Hi All,

I am not fully technical but I am learning a bit using Laragon.

Can someone please guide on steps required to migrate from SQL to Mariadb and move existing projects?

I found this link https://www.kreaweb.be/laragon-migrate-mariadb/ but I am stuck on exporting and importing existing projects bit which has not been explained on this page.

Please help.

@Pen-y-Fan Can you please make one of your excellent Laragon guides on this? I am sure all would appreciate it, including me! 

Thanks

**Comment by Pen-y-Fan** [comment, marked as answer]
I would recommend HeidiSQL, to export and import:

- Login to the MySQL database
- Right-click the database and select Export database as SQL

<img width="197" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/3c99fc30-ea73-45a2-bc40-80275a92135b">


- Export the data as per this screenshot:

<img width="659" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/8e4ebf1a-fe6b-4344-9329-f9cde10b5615">

- Login to the new MariaDB server
-  Select File > Run SQL file...

<img width="144" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/c18f3a74-46cd-47b3-a4d8-4e9aed7f7f86">

- Select the file you exported and it will auto-run
- if you have problems, to troubleshoot, use: File > Load SQL file

**Comment by KoolPal** [comment]
Thank you!
